scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/InfoChain.java

```diff
@@ -0,0 +1,43 @@
+
+package org.scijava.ops.api;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class InfoChain {
```

Comment:
scijava/scijava#81

Reply:
> Chain implies linear. This is actually a tree. So I'd name it `OpTree`.

@ctrueden I like `OpInfoTree`, or maybe just `InfoTree`. We shouldn't name it `OpTree`, because it has no Ops. 

> this tree is a hierarchy of _resolved_ dependencies, right?

I'm not sure what you mean by this. If you mean that each `OpDependency` is represented by an `OpInfo` in the right place, then yeah, it is.

I added some javadoc to `InfoChain` in f7cd642c8de8b818498ed06bb6725da3e0faa93d.